packages/demo/package.json

```diff
@@ -7,7 +7,6 @@
     "dev": "vite",
     "build": "vite build",
     "lint": "eslint .",
-    "test": "jest",
```

Comment:
hhhh, both the `demo` and `sandbox` packages don't have tests, so no need to keep it here, right? 🤔

Reply:
yes, that's why i removed them :)